openapi/adintel/adintel.yaml

```diff
@@ -341,6 +441,40 @@ components:
         - id
         - type
         - attributes
+    campaignInfo:
+      title: campaignInfo
+      type: object
+      x-examples: {}
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+        Information about an advertising campaign.  Requires the Advanced Reporting Addon to be active on the Business Location to access
+      x-lifecycle:
+        status: trustedTester
+      x-tags:
+        - accountStats
+      properties:
+        id:
+          type: string
+          description: The resource ID of the Connected Advertising Account
+          example: 'AG-HDV4JKHW:google:7531238654:94749174'
+        type:
+          type: string
+          description: campaignInfo
+        attributes:
+          type: object
+          properties:
+            name:
+              type: string
+              description: Name of the Campaign
+            isConnected:
+              type: boolean
+              description: Flag indicating if the campaign is connected to the Business Location
```

Comment:
Should this be an emun matching `connectedAccounts.statusCode`. Then your query param filter can be updated to use an enum with a list of options to include.

Reply:
It doesn't quite match the status on a connected account, because one represents the existence of a vstore entry, and the other represents the state of an auth token